doc/markdown/user/implementation/mapping/CUDA.md

```diff
@@ -143,7 +143,7 @@ The following tables list the functions available in the [CUDA Runtime API](http
 |cudaGetMipmappedArrayLevel|-|
 |cudaGetSymbolAddress|-|
 |cudaGetSymbolSize|-|
-|cudaHostAlloc|n/a|
+|cudaHostAlloc|n/a, the existing buffer can be pinned using alpaka::mem::buf::traits::PrepareForAsyncCopy<TBuffer>::prepareForAsyncCopy(buffer)|
```

Comment:
Wait, where did my template parameter go? I guess some conflict between the C++ <> and formatting.

Reply:
We should mention the public `alpaka::mem::buf::prepareForAsyncCopy(buffer)` trait accessor instead of directly the specific implementation.

Reply:
Good point, fixed. Also renamed the parameter to `memBuf` to match the other names in this table.